MkDocs/contribute/docs/prereqs.md

```diff
@@ -2,45 +2,79 @@
 
 Here are the required pre-requisites for contributing to The Maximo labs.
 
-## Setup Python 3
+## Install and Setup Python 3
+
+Even though Python 3.8.2 is installed on your Mac, you'd want to use the latest release.</br>
+First install Homebrew on Mac as described at [https://brew.sh/](https://brew.sh/):
+
+    $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+![Install Homebrew](/img/contribute/pre-req_1.png)
+
+Press RETURN. When completed it is time to install the latest version of Python 3:
+
+    $ brew install python3
+
+Quit and restart the Terminal to have your PATH updated, then run python3 -V and pip3 -V:
+![Homebrew installation of Python3](/img/contribute/pre-req_2.png)
+
+### Uninstall Python 3
+
+If you want to uninstall Python 3 and Homebrew:
+
+    $ brew uninstall python3
+    $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
+
+!!! note "Additional Uninstallation Information"
+    Further information about uninstalling:</br>
+    [https://huybien.com/how-to-completely-uninstall-python-on-macos/](https://huybien.com/how-to-completely-uninstall-python-on-macos/)</br>
+    [https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew](https://docs.brew.sh/FAQ#how-do-i-uninstall-homebrew)
+
+
+## Install MkDocs
 
 You can install Python packages with
 
-    pip3 install <package>
+    $ pip3 install <package>
 
-This will install into the site-package directory
 
-    /usr/local/lib/python3.7/site-packages
```

Comment:
While your changes to add some value here it makes it so that the bare minimum net net the user needs to accomplish the task is cluttered.   For example  put virtual Env as a pre-requiste and link to the doc somewhere earlier in this section like we do with the labs.  So when they get to this section it is net net commands is my suggestion.

Reply:
Thanks for the chat on Slack - I'll tidy it up a bit.